src/main/java/com/monov/cloud/gateway/dto/StudentOrCourseIds.java

```diff
@@ -0,0 +1,16 @@
+package com.monov.cloud.gateway.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class StudentOrCourseIds {
```

Comment:
Seems like you already have this model inside a different package?

Reply:
Fixes will be applied in DTOs branch's PR